x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/_shard_allocation.scss

```diff
@@ -14,52 +14,57 @@ monitoring-shard-allocation {
   cluster-view {
     display: block;
   }
-  .parent {
-    padding-top: 14px;
-    border-left: 3px solid $euiColorSuccess !important;
-    &.red {
-      border-left: 3px solid $euiColorDanger !important;
-    }
-    &.yellow {
-      border-left: 3px solid $euiColorWarning !important;
-    }
-  }
-  td.unassigned {
+
+  .monUnassigned {
     vertical-align: middle;
     width: 150px;
   }
 
-  .child {
+  .monChildren {
+    padding-top: $euiSizeL;
+  }
+
+  .monChild {
     float: left;
     align-self: center;
+    background-color: $euiColorLightestShade;
+    margin: $euiSizeS;
+    border: 1px solid $euiColorMediumShade;
+    border-radius: $euiSizeXS;
+    padding: $euiSizeXS/2 0;
 
     &.index {
```

Comment:
This classname is too generic. Needs to follow the prefixing. Specifically these look like modifying elements so should likely be `--`